src/simplewallet/simplewallet.cpp

```diff
@@ -362,6 +362,29 @@ bool simple_wallet::seed_set_language(const std::vector<std::string> &args/* = s
   return true;
 }
 
+bool simple_wallet::change_password(const std::vector<std::string> &args)
+{
+  // watch-only wallets don't need passwords, right?
+  if(m_wallet->watch_only())
```

Comment:
Watch only wallets can have a password. Indeed, when you create one with save_watch_only, you've asked for one.